includes/html/api_functions.inc.php

```diff
@@ -517,11 +518,11 @@ function list_bgp(\Illuminate\Http\Request $request)
     }
     if (!empty($local_address)) {
         $sql .= ' AND `bgpPeers`.`bgpLocalAddr` = ?';
-        $sql_params[] = $local_address;
+        $sql_params[] = IP::parse($local_address)->uncompressed();
```

Comment:
This will throw an exception if it isn't a valid IPv6 address

Reply:
best to put a try catch block around it and send an error to the client.

Reply:
Fixed :)